doc/docs/master/deploy-manage/deploy/deploy-pachyderm-jupyterhub.md

```diff
@@ -0,0 +1,245 @@
+# Deploy Pachyderm with JupyterHub
```

Comment:
I think it'd help for us to diagram out what this looks like. Something like this, but hopefully more polished:

<img width="1008" alt="Screen Shot 2020-01-06 at 2 01 58 PM" src="https://user-images.githubusercontent.com/127386/71841329-0ccaff00-30b7-11ea-964c-6a6b626d9dda.png">

Reply:
added